compute-hypothetical-feature.mjs

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env node
+import {readFileSync} from 'node:fs';
+
+import {compare} from 'compare-versions';
```

Comment:
This isn't strictly needed, but `historical-browser-data.json` doesn't have release dates, so it avoid having to do a lookup table from browser version to release date. That also avoids any problems around version formats changing slightly over time, which would lead to holes in that lookup table. https://github.com/Fyrd/caniuse/commit/3d689eeffc58cf85fceb6d1c25f10da8cbe0f813 is an example of that happening.